test/sql/include/append_query.sql

```diff
@@ -346,3 +346,15 @@ LEFT JOIN i2661 ht ON
   (FLOOR(EXTRACT (EPOCH FROM ht."timestamp") / 300) * 300 = EXTRACT (EPOCH FROM ts.timestamp))
   AND ht.timestamp > '2019-12-30T00:00:00Z'::timestamp;
 
+-- #3030 test chunkappend keeps pathkeys when subpath is append
+-- on PG11 this will not use ChunkAppend but MergeAppend
+SET random_page_cost=1;
```

Comment:
I think it it better to just enable_indexscan and disable seqscan. The intent is clearer.
Need to reset at the end of this test case.

Reply:
changed